modules/exploits/windows/browser/ms14_064_ole_code_execution.rb

```diff
@@ -358,6 +359,11 @@ def get_html
   end
 
   def on_request_exploit(cli, request, target_info)
+    if get_target.name.match(OperatingSystems::Match::WINDOWS_7) && !datastore['AllowPowershellPrompt']
```

Comment:
I'm asking because the module description says: `Internet Explorer 3.0 until version 11 within Windows 95 up to Windows 10`

Am I misunderstanding?